src/index.html

```diff
@@ -2,16 +2,176 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>
+      HTML Form
+    </title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>
+          Personal information
+        </legend>
+        <label class="form_field">
+          Surname:
+          <input 
+        type="text" 
+        name="surname"
+        autocomplete="off"
+        required
+          >
```

Comment:
```suggestion
          <input 
            type="text" 
            name="surname"
            autocomplete="off"
            required
          >
```
Fix all cases
![image](https://user-images.githubusercontent.com/96409563/222079956-da72d29e-9788-4d8f-bf6f-d4d9f98f0272.png)